app/main.py

```diff
@@ -2,0 +1,20 @@
+import datetime
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    num_maskless = 0
+    for friend in friends:
+        if "vaccine" not in friend:
+            return "All friends should be vaccinated"
+        expiration_date = friend["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            return "All friends should be vaccinated"
```

Comment:
Code Style: Similarly, the error message here could be more descriptive. Instead of saying 'All friends should be vaccinated', consider something like 'All vaccinations should be up to date.' This would make it clearer what the actual problem is.